README.md

```diff
@@ -84,6 +86,66 @@ or have `InfoDialog` in their behavior text.
 Any option after the double dash (`--`) gets sent to `mocha`. Check out
 [mocha's usage](https://mochajs.org/#usage) for ideas.
 
+### Flow
+
+There is limited support for using [Flow](https://flowtype.org/)
+to check for problems in the source code.
+
+Here is how to develop with constant feedback about Flow issues:
```

Comment:
I would rephrase this to `To check for flow issues during development on any file change, run:`